Add :pulledpercent: replacement

Cy4n1d3 opened the pull request:
This is a quite simple addition which just adds another custom replacement to the options:
`:pulledpercent:` can be used to display the trash percentage including the current pull.
I've also increased the count for "Show Forces Glow" in demo mode, so it nets 101% (which might be used to see how overcount looks like).

Let me know if any further changes are needed!

happenslol commented on Options.lua:
```diff
@@ -217,12 +217,14 @@ function WarpDeplete:InitOptions()
             desc = L["Choose how your forces progress will be displayed"],
             sorting = {
               ":percent:",
+              ":pulledpercent:",
```
I think I would prefer something like `percentafterpull` here, since we already have `remainingpercentafterpull`. Regretting that I didn't put any word separators or capitalization in the naming here now, but I suppose what's done is done...

happenslol commented on Util.lua:
```diff
@@ -10,8 +10,8 @@ function Util.formatForcesText(
-  local currentPercent = (currentCount / totalCount) * 100
-  if currentPercent > 100.0 then currentPercent = 100.0 end
+  local currentPercent = math.min((currentCount / totalCount) * 100, 100.0)
+  local currentPulledPercent = (currentCount / totalCount) * 100
```
Hmm, so if I understand this correctly, this will show above 100 if the current pull exceeds 100 percent? I'm not sure how I feel about that behavior, since that's basically an additional "hidden feature" of this format string, for people that want to show how much they went over 100...

Maybe it would be better to add a general flag that allows showing when you're over 100 for any format string, and keep this the same as `currentPercent`?

That would simplify this line to `local currentPulledPercent = currentPercent`, and below you could just do

```lua
currentPulledPercent = math.min(currentPercent + pulledPercent, 100.0)
```

Then, if you want to add a flag for showing 100+, you can pull the `math.min` out into a function that would be called something like `clampPercent`, which can check if the flag is set and return the value unchanged or clamp it.

Cy4n1d3 replied:
Yea, I was (or am) quite unsure about that as well. I could actually just revert to one of my older commits (from git reflog) where the code was more or less exactly like that (if the current pull would bring the percentage above 100, it is floored).
Also thought about adding an option for that (but didn't want to bring the discussion into this MR) so I might just go ahead and implement that right away.
Would an option toggle like "Limit forces count shown to 100%" be fine for you? I'm open for better naming options! :)

Cy4n1d3 replied:
Please have another look at the updated code and feel free to request any changes.
Thanks for your input!

happenslol reviewed (changes requested):
Thanks for your contribution! Looks great overall, I just have 2 suggestions. Let me know what you think!

Cy4n1d3 commented:
I've pushed a few changes:
- new setting to allow showing of unclamped Forces Percentage
- small change to `UpdateDemoModeForces()` to show percentages based on "Show Forces Glow" setting (which takes preference if set)
- renamed `:pulledpercent:` placeholder to `:percentafterpull:`
- refactored generation of Forces Text to account for the new setting and the new placeholder, keeping old behaviour intact
- moved string replacement for `:percentafterpull:` below `pullCount > 0` condition

happenslol commented:
Awesome, thanks for your changes! Will include this in the next release :-)
